common-rest/src/test/java/org/uniprot/api/rest/validation/QueryFieldValidatorTest.java

```diff
@@ -254,6 +256,40 @@ void isValidInvalidFieldWithForwardSlashReturnFalse() {
         assertEquals("invalid", validator.getErrorFields(ErrorType.FIELD).get(0));
     }
 
+    @ParameterizedTest
+    @ValueSource(
+            strings = {
```

Comment:
Can you also please test prefixes for these IDs as well?

whitelist.field.uniprotkb.ecocyc=^[0-9A-Za-z-]+$
whitelist.field.uniprotkb.hgnc=^[0-9]+$
whitelist.field.uniprotkb.hostdb=^[0-9A-Za-z-]+$
whitelist.field.uniprotkb.lncrna=^[0-9A-Za-z]+$
whitelist.field.uniprotkb.metacyc=^[0-9A-Za-z-]+$
whitelist.field.uniprotkb.mrna=^[0-9A-Za-z_]+$
whitelist.field.uniprotkb.pac=^[0-9]+$
whitelist.field.uniprotkb.pr=(?i)([OPQ][0-9][A-Z0-9]{3}[0-9]|[A-NR-Z]([0-9][A-Z][A-Z0-9]{2}){1,2}[0-9])(-[0-9]+)?
whitelist.field.uniprotkb.slp=^[0-9]{9}$
whitelist.field.uniprotkb.vectorbase=^[0-9A-Z]+$